applications/h264/h264_endoscopy_tool_tracking/main.cpp

```diff
@@ -43,18 +41,17 @@ class App : public holoscan::Application {
     int64_t source_block_size = width * height * 3 * 4;
     int64_t source_num_blocks = 2;
 
-    auto bitstream_reader =
```

Comment:
I applied clang-format to the modified files which accounts for the style-related changes 

(similarly `black` was applied to Python scripts)